Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Don't forget the semi-colon to get rid of the MPL output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>